langchain/client/models.py

```diff
@@ -97,3 +105,57 @@ def validate_time_range(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         if start_time and end_time and start_time > end_time:
             raise ValueError("start_time must be <= end_time")
         return values
+
+
+class APIFeedbackSource(BaseModel):
+    """API feedback source."""
+
+    type: ClassVar[str] = "api"
+    metadata: Optional[Dict[str, Any]] = None
+
+
+class FeedbackBase(BaseModel):
+    """Feedback schema."""
+
+    created_at: datetime = Field(default_factory=datetime.utcnow)
+    """The time the feedback was created."""
+    modified_at: datetime = Field(default_factory=datetime.utcnow)
```

Comment:
style: i like `updated_at` better (sounds better to me when contrasted with `created_at`)

Reply:
Ya that may be, but we already use modified_at on other resources and I would rather be consistent with that